server/http_handler_test.go

```diff
@@ -443,7 +441,7 @@ func (ts *HTTPHandlerTestSuite) TestGetTableMVCC(c *C) {
 	info := data.Value.Info
 	c.Assert(info, NotNil)
 	c.Assert(len(info.Writes), Greater, 0)
-	startTs := info.Writes[2].StartTs
```

Comment:
`info.Writes` returned by mocktikv is
```
type:Lock start_ts:416637082071465985 commit_ts:416637082071728128 
type:Lock start_ts:416637082066485249 commit_ts:416637082066485250 
start_ts:416637082062553091 commit_ts:416637082063077376 short_value:"\200\000\001\000\000\000\002\001\0002" 
start_ts:416637082062553089 commit_ts:416637082062553090 short_value:"\200\000\001\000\000\000\002\001\0001"
```

by unistore is
```
start_ts:416637064772321280 commit_ts:416637064772583424 short_value:"\200\000\001\000\000\000\002\001\0002" 
start_ts:416637064772059142 commit_ts:416637064772059143 short_value:"\200\000\001\000\000\000\002\001\0001"
```

Is this a bug in unistore @cfzjywxk

Reply:
Seems the `Op_lock` is not returned in `unistore`, could change the case  temporarily  to
```
startTs := info.Writes[0].StartTs
```
to let it pass @bobotu